src/components/Calculator.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import './Calculator.css';
+
+const Calculator = () => {
+  const [displayValue] = useState('0');
+
+  const handleButtonClick = () => {
+
+  };
+
+  const buttonRows = [
+    ['AC', '+/-', '%', '÷'],
+    ['7', '8', '9', '×'],
+    ['4', '5', '6', '-'],
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
+  return (
+    <div className="calculator">
+      <div className="display">{displayValue}</div>
+      <div className="buttons">
+        {buttonRows.map((row, rowIndex) => (
+          <div className="row" key={uuidv4()}>
+            {row.map((button, columnIndex) => {
```

Comment:
Great job so far! 👏 However, for the calculator component, we need to have at least one child component. 🧒🔢 Could you please implement that? Thanks! 😊